src/compliance/privacy/gdpr.md

```diff
@@ -0,0 +1,28 @@
+---
+group: compliance
+title: General Data Protection Regulation
+redirect_from:
+  - /guides/v2.3/architecture/gdpr/magento-2x.html
+  - /guides/v2.3/architecture/gdpr/magento-1x.html
+  - /guides/v2.2/architecture/gdpr/magento-2x.html
+  - /guides/v2.2/architecture/gdpr/magento-1x.html
+---
+
+The European Union (EU) enacted [General Data Protection Regulation](https://ec.europa.eu/info/law/law-topic/data-protection_en) (GDPR) to give its citizens more control over their personal data. GDPR applies to any organization operating within the EU. It also applies to organizations outside of the EU that offer goods or services to customers or businesses in the EU.
+
+System integrators can use the data flow diagrams and database information in the Personal Information Reference to build scripts to resolve use cases similar to the following:
+
+-  A shopper asks for a copy of the data the merchant has stored about her
+-  A shopper requests that all information about him be deleted
+
+## Personal Information Reference
+
+For technical information, see the data flow diagrams and database entity mappings in the Personal Information Reference that applies to the version(s) of Magento that you support.
+
+-  [Personal Information Reference (Magento 2.x)]({% link compliance/privacy/pi-data-reference-m2.md -%})
+-  [Personal Information Reference Magento 1.x)]({% link compliance/privacy/pi-data-reference-m1.md -%})
```

Comment:
```suggestion
-  [Personal Information Reference Magento 1.x)]({% link compliance/privacy/pi-data-reference-m1.md %})
```

Reply:
I think the GitHub editor is adding that <span class="x x-first x-last">-</span>, because it's not in my code.

Reply:
Fixed.